src/routes/alerts/alerts.controller.spec.ts

```diff
@@ -178,12 +178,13 @@ describe('Alerts (Unit)', () => {
         );
 
         networkService.get.mockImplementation((url) => {
+          console.log(url);
```

Comment:
`console.log` left here 😄

Reply:
🙈 Removed in 8469ed0a094b62c49b0ca23355456cc35c32e3ad.